src/main/scala/uk/sky/fs2/kafka/topicloader/TopicLoader.scala

```diff
@@ -22,9 +22,12 @@ object TopicLoader extends TopicLoader {
   private[topicloader] given [K, V]: Show[ConsumerRecord[K, V]] =
     Show.show(cr => s"${cr.topic}-${cr.partition}")
 
+  private case class PartitionLast(topicPartition: TopicPartition, offset: Long)
```

Comment:
`PartitionLastOffset` might make this clearer?

Reply:
[`2517fa4` (#64)](https://github.com/sky-uk/fs2-kafka-topic-loader/pull/64/commits/2517fa45550c29836db0dda52735ac29b58301f8)